src/App.js

```diff
@@ -1,13 +1,46 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+} from 'react-router-dom';
 import Calculator from './components/calculator';
+import Home from './components/home';
+import Quote from './components/quote';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] OPTIONAL 
You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.